src/open_inwoner/pdc/models/product.py

```diff
@@ -232,7 +230,7 @@ def get_absolute_url(self, category=None):
         )
 
     def has_cta_tag(self):
-        return "\[CTABUTTON\]" in self.content
+        return "\[CTABUTTON\]" in self.content  # noqa
```

Comment:
Why is this `noqa`?

Reply:
Due to `src/open_inwoner/pdc/tests/test_product.py:271:46: W605 invalid escape sequence '\]'`

Reply:
I wonder what this is supposed to be: does it even need the backslashes or was this a remnant of a regex?

Reply:
I'm not sure, but I didn't want to change the code without knowing whether it's still necessary